homeassistant/components/light/strings.json

```diff
@@ -323,10 +323,6 @@
         "transition": {
           "name": "[%key:component::light::services::turn_on::fields::transition::name%]",
           "description": "[%key:component::light::services::turn_on::fields::transition::description%]"
-        },
-        "flash": {
-          "name": "[%key:component::light::services::turn_on::fields::flash::name%]",
-          "description": "[%key:component::light::services::turn_on::fields::flash::description%]"
```

Comment:
According to `services.yaml` `turn_off` has this as an option so removing it would be incorrect

Reply:
Confirmed, flash does indeed exist.  I've commited a change to revert this particular part :)